web/src/components/tasks/environment-questionnaire/results/ResultsSectionAccordion.tsx

```diff
@@ -11,7 +11,7 @@ interface Props {
   isDividerDisabled?: boolean;
 }
```

Comment:
sand - I don't think those props above (specifically mini and isDividerDisabled are actually used ever?)